src/involvement/models/role.py

```diff
@@ -17,6 +21,20 @@ class Meta:
         verbose_name = _('Role')
         verbose_name_plural = _('Roles')
         default_permissions = ()
+        permissions = (
+            ('admin', _('Admin')),
+            ('fum', _('FUM')),
+            ('board', _('Board')),
+            ('bureau', _('Bureau')),
+            ('group_leader', _('Group Leader')),
+            ('engaged', _('Engaged')),
+        )
```

Comment:
Maybe they are not supposed to be in the new system, but I'm primarily missing the Election committee. Are they no longer in the system.

Reply:
No, the election committee has no more use. Every role except engaged can approve applications for some roles with lower level.